src/mono/netcore/System.Private.CoreLib/src/ILLink/System.Private.CoreLib.xml

```diff
@@ -757,9 +750,5 @@
 		<type fullname="System.Threading.ThreadStartException">
 			<method name=".ctor" />
 		</type>
-		<type fullname="System.Runtime.InteropServices.Marshal">
-			<!-- marshal-ilgen.c: emit_marshal_custom_get_instance -->
-			<method name="GetCustomMarshalerInstance" />
```

Comment:
this is used too:

https://github.com/dotnet/runtime/blob/fbe4d10539a60f3a536ade5cdce991ae09fa9b84/src/mono/mono/metadata/marshal-ilgen.c#L4479

Reply:
This was added above to the existing Marshal type block.

Reply:
Yep, it's still there https://github.com/dotnet/runtime/pull/38442/files#diff-621abfb605caafcaac53649d9dde3cc0R661